### ISSUE TITLE
RTE editing selected media to update alt text

### ISSUE THREAD

**Issue by alina-tincas** [umbraco/Umbraco-CMS#20066]
### Which Umbraco version are you using?

16.1.1

### Bug summary

Compared to v13, it seems that on v16 on RTE when re-opening an image to edit the alt text, it is missing the modal to edit the alt text. This means that currently on v16 you will need to re-pick the same image and choose it so you can get again to the "edit selected media" modal to edit the alt text. However on v13 it was enough to just selecting the image and opening the media option to get the "edit selected media".

https://github.com/user-attachments/assets/2d2b6313-4481-4c67-ba1f-2148fbcc5a44

### Specifics

There are no errors in console when re-picking the image in RTE. I have tried this on v15 as well and it is the same. So I am not sure if perhaps this was not implemented yet from v14, or if this is the default behaviour with the new TipTap RTE.

### Steps to reproduce

**1.** On a v13 umbraco installation add a doc type with the RTE (tipTap) property. Then add an image, and afterwards try to edit it again so you can edit the alt text. Notice how a pop-up/modal opens automatically to edit the selected media, more specifically the alt and caption texts.

<img width="825" height="816" alt="Image" src="https://github.com/user-attachments/assets/1490d126-946d-47f6-a049-f615fb525558" />

**2.** Try the same steps on a v16 umbraco installation, and notice that the modal does not open automatically. You will need to choose the same image again to get to the edit the selected media modal.

### Expected result / actual result

When you click on the media option from RTE, you will get the option to edit the selected media.

<img width="423" height="458" alt="Image" src="https://github.com/user-attachments/assets/f7f86684-4bdd-4ecf-9793-b661855a50fa" />

---
_This item has been added to our backlog AB#56331_
